rust/list/src/items.rs

```diff
@@ -45,6 +51,65 @@ pub struct Item {
     pub labels: Vec<Label>,
 }
 
+impl From<ItemC> for Item {
+    fn from(item_c: ItemC) -> Self {
+        Item {
+            id: None,
+            uuid: Uuid::nil(),
+            name: c_char_to_string(item_c.name),
+            due_date: None,
+            completion_date: None,
+            labels: vec![]
+        }
+    }
+}
+
+impl From<Item> for ItemC {
+    fn from(item: Item) -> Self {
+        ItemC {
+            name: string_to_c_char(item.name.clone())
+        }
+    }
+}
+
+// Wrapper structs so that we can have nice From traits.
+// We can only implement Traits for structs that we own.
+pub struct Items {
+    pub vec: Vec<Item>
+}
+
+impl Items {
+    pub fn new(vec: Vec<Item>) -> Items {
+        Items {
+            vec: vec
+        }
+    }
+}
+
+pub struct ItemsC {
```

Comment:
I am not sure why we need a struct to wrap a vector of items. Why can't we just pass the `vec` around and do?

```
impl From<Vec<Item>> for Vec<ItemC>
```